docker/docker-compose.yaml

```diff
@@ -2,9 +2,9 @@ version: "3"
 services:
   controller:
     image: vc-authn-oidc-controller
-    depends_on:
-      controller-db:
-        condition: service_started
+    # depends_on:
+    #   controller-db:
+    #     condition: service_started
```

Comment:
Do you mind removing this completely? It appears that we keep commenting/uncommenting these lines 😅